packages/dht/src/connection/WebRTC/BrowserWebRtcConnection.ts

```diff
@@ -0,0 +1,217 @@
+import EventEmitter from "events"
+import { Event, IWebRtcConnection, RtcDescription } from "./IWebRtcConnection"
+import { IConnection, Event as ConnectionEvent, ConnectionType } from "../IConnection"
+import { PeerDescriptor } from "../../proto/DhtRpc"
+import { ConnectionID } from "../../types"
+import { Logger } from '../../helpers/Logger'
+import { IWebRtcCleanUp } from './IWebRtcCleanUp'
+
+enum ConnectionState { CONNECTING = 'connecting', OPEN = 'open', CLOSED = 'closed' }
+
+const logger = new Logger(module)
+
+export const WebRtcCleanUp = new class implements IWebRtcCleanUp {
+    cleanUp(): void {
+    }
+}
+
+export class NodeWebRtcConnection extends EventEmitter implements IWebRtcConnection, IConnection {
+
+    public connectionId: ConnectionID = new ConnectionID()
+    public readonly connectionType: ConnectionType = ConnectionType.WEBRTC
+
+    private lastState: ConnectionState = ConnectionState.CONNECTING
+    private stunUrls = ['stun:stun.l.google.com:19302']
+    private peerConnection: RTCPeerConnection | null = null
```

Comment:
Slightly more readable if we use TypeScript optionality. Usually I prefer `undefined` instead of `null` because TypeScript has this better support for it. (And to avoid having multiple types for missing values)

```suggestion
    private peerConnection?: RTCPeerConnection
```